templates/sonar.properties

```diff
@@ -271,18 +402,23 @@ sonar.search.port={{ sonar_search_port }}
 # Paths to persistent data files (embedded database and search index) and temporary files.
 # Can be absolute or relative to installation directory.
 # Defaults are respectively <installation home>/data and <installation home>/temp
-sonar.path.data={{ sonar_path_data }}
+#sonar.path.data=data
 #sonar.path.temp=temp
 
+# Telemetry - Share anonymous SonarQube statistics
+# By sharing anonymous SonarQube statistics, you help us understand how SonarQube is used so we can improve the product to work even better for you.
+# We don't collect source code or IP addresses. And we don't share the data with anyone else.
+# To see an example of the data shared: login as a global administrator, call the WS api/system/info and check the Statistics field.
+#sonar.telemetry.enable=true
 
 
 #--------------------------------------------------------------------------------------------------
 # DEVELOPMENT - only for developers
+# The following properties MUST NOT be used in production environments.
 
-# Dev mode allows to reload web sources on changes and to restart server when new versions
-# of plugins are deployed.
-#sonar.web.dev=false
+# Elasticsearch HTTP connector
+#sonar.search.httpPort=-1
 
-# Path to webapp sources for hot-reloading of Ruby on Rails, JS and CSS (only core,
-# plugins not supported).
-#sonar.web.dev.sources=/path/to/server/sonar-web/src/main/webapp
+#--------------------------------------------------------------------------------------------------
+# SECRETS
+sonar.secretKeyPath=/opt/sonar/.sonar/sonar-secret.txt
```

Comment:
Can you use the same variable you used in the task to populate this file? `sonar-secretKeyPath={{ sonar_home }}/.sonar/sonar-secret.txt`

Reply:
Yes!

Reply:
Good suggestion, and good catch